gnpy/core/elements.py

```diff
@@ -224,37 +226,15 @@ def __call__(self, spectral_info):
         pref = self.update_pref(spectral_info.pref)
         return spectral_info._replace(carriers=carriers, pref=pref)
 
-FiberParams = namedtuple('FiberParams', 'type_variety length loss_coef length_units \
-                                         att_in con_in con_out dispersion gamma')
 
 class Fiber(Node):
     def __init__(self, *args, params=None, **kwargs):
-        if params is None:
+        if not params:
```

Comment:
None, empty lists, dictionaries and tuples are interpreted as False under boolean conditions. In this case, the if condition is used to handle the exception within the FiberParams object instead of raising an Exception when the ** unpacker is called.